### DISCUSSION TITLE
Error in multiselect

### DISCUSSION THREAD

**Discussion by furrutia95** [mdbootstrap/TW-Elements#1974] [Support from community]
Hi, im new in Tailwind Elements and i want to trying making a multiselect, but when i use the Docs:
  //Logica MultiSelect
  const multiSelect = document.querySelector("#multiSelect");
  const multiSelectInstance = Select.getInstance(multiSelect);
  multiSelectInstance.setValue("4");
  
  I have this trouble 
  
![image](https://github.com/mdbootstrap/Tailwind-Elements/assets/54600755/13695988-b2cb-4f69-9c65-764290363bbf)

Anyone can help me plz? I need solve this from my work :( 
Edit: It used to work, but out of nowhere it started to fail.

**Comment by juujisai** [comment]
Hi! Was the component initialized? Did you call the `initTE` method / use the `via JS` initialization? You can also change the `getInstance` method to `getOrCreateInstance`.
